### DISCUSSION TITLE
Should cities be swipable by default on main screen?

### DISCUSSION THREAD

**Discussion by papjul** [breezy-weather/breezy-weather#183] [Polls]
Following #168, it seems the horizontal swipe is a little sensitive on some devices, although it doesn’t seem to reproduce for everyone.

In either case, having two horizontal swipe events on the same page is probably not a good idea from a user experience point of view, as if you miss a little the safe zone of swipe, you will end up triggering the wrong swipe.

So I was wondering if this “swipe between cities” is something that you feel all users should have by default? Personally, when I have two cities, swipe can be useful, however if I have more than 3, when I swipe, I never know on which city I will end up, so it’s quicker for me to just go to the location list. On the other hand, the “bookmark icon” is not the most accessible thing on top of main screen, and can even be hidden after you scrolled a bit.

Small note on tablet users: they probably don’t need swipe as by default, they have the location list on the left as a drawer layout.

While answering the poll, think about what would be the best experience for “most users”, not about yourself (now that you know that the setting will exist, you know that whatever is the result of this poll, you can always change it in the settings).

**Comment by min7-i** [comment]
I'm leaning more towards disabling the "location swipe" by default. In my opinion the feature is rather hidden. As you already pointed out there are other swipe actions within the cards that, I feel, are more apparent / intuitive. However, there is no indicator of any kind that you can do another horizontal swipe to switch between locations. I'd say new users discover the location swipe rather by accident.

Either way questions could come up, I guess. Users coming from Geometric Weather might expect swiping to work and wonder what happened if it's disabled by default while new users will probably not expect that feature to be available and create issues like #168 if it's enabled by default.

**Comment by papjul** [comment]
Given results are mixed, I had a look at other weather apps, and I came up with the idea of increasing required swipe distance to go to next location from 20 % of screen width to 50 %. It will be available in v4.6.0.
